src/components/common/program-section-templates/single-title-quintuple-description/SingleTitleQuintupleDescription.module.scss

```
@@ -22,11 +22,108 @@ $desc-input-height: 115px;
     aspect-ratio: 1440 / 650;
 }
 
+.template-layout {
+    height: 100%;
+    width: 100%;
+    max-width: 1320px;
+    margin: 0 auto;
+    box-sizing: border-box;
+
+    display: grid;
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)